test/FlapperUniV2.t.sol

```diff
@@ -170,29 +197,23 @@ contract FlapperUniV2Test is DssTest {
             pair:     pair,
             receiver: PAUSE_PROXY,
             swapOnly: false
-        });
-        _flapper = FlapperUniV2(flapperInstance.flapper);
+        }));
 
         // Note - this part emulates the spell initialization
         vm.startPrank(PAUSE_PROXY);
         FlapperUniV2Config memory cfg = FlapperUniV2Config({
-            hop  : 30 minutes,
-            want : WAD * 97 / 100,
-            pip  : address(_medianizer),
-            hump : 50_000_000 * RAD,
-            bump : 5707 * RAD,
-            caller: address(vow),
-            chainlogKey: "MCD_FLAP_LP"
+            want:        WAD * 97 / 100,
+            pip:         address(_medianizer),
+            pair:        pair,
+            splitter:    address(splitter),
+            chainlogKey: "MCD_FLAP_BURN"
```

Comment:
This should be "MCD_FLAP_LP".

Reply:
Hmm, not sure. The way I saw it, `MCD_FLAP_BURN` indicates the burner strategy, which could be "swap and burn" or "swap and LP". Maybe the key should be renamed differently to prevent confusion with the specific "swap and burn" strategy though. What about "MCD_FLAP_STRAT" ?

Reply:
Not sure I follow, this test is for the "swap and LP" flapper.

Reply:
Actually nevermind, I thought of reusing the same `MCD_FLAP_BURN` key for both flappers (burners) but since they will probably coexist simultaneously, it's best to give them each their own key.